third_party/blink/renderer/core/farbling/brave_session_cache.cc

```diff
@@ -177,6 +176,7 @@ int FarbledPointerScreenCoordinate(const DOMWindow* view,
 
 BraveSessionCache::BraveSessionCache(ExecutionContext& context)
     : Supplement<ExecutionContext>(context) {
+  default_language_ = blink::DefaultLanguage().GetString().Left(2);
```

Comment:
can this change during the lifetime of an ExecutionContext? I'm guessing not, but still might be better not to cache it locally?

Reply:
removed in https://github.com/brave/brave-core/pull/14832/commits/3fc1d09601b824f7a2640554d66204f371a71169